### PR TITLE
Update Changelog  [ci skip]

### PR DESCRIPTION
**Pull request by CorainChicago**
Update for the Changelog for merged pull requests from #1117 - 1384 with the label CHANGELOG MISSING

### CODE REVIEW

**Comment by CorainChicago** [comment]
This should bring the Changelog up-to-date for the merged and closed items with the Changelog Missing label.  Please review and let me know if items need to be moved or attributed to more than one contributor.

**Comment by bf4** [comment]
:100: will review, thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## 0.10.x
 
+
```
Comment (bf4) [inline CHANGELOG.md]: :scissors:

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,8 +33,15 @@ Features:
         e.g. `has_many :reviews, virtual_value: [{ id: 1 }, { id: 2 }]`
   * Removes dynamically defined methods on the serializer
 - [#1336](https://github.com/rails-api/active_model_serializers/pull/1336) Added support for Grape >= 0.13, < 1.0 (@johnhamelink)
+- [#1322](https://github.com/rails-api/active_model_serializers/pull/1322) Instrumenting rendering of resources (@bf4)
```
Comment (bf4) [inline CHANGELOG.md]: include `@maurogeorge`

File: CHANGELOG.md [hunk 3]
```diff
@@ -32,8 +33,15 @@ Features:
...
 - [#1291](https://github.com/rails-api/active_model_serializers/pull/1291) Add logging (@maurogeorge)
+- [#1272](https://github.com/rails-api/active_model_serializers/pull/1272) Add PORO serializable base class: ActiveModelSerializers::Model (@bf4)
+- [#1255](https://github.com/rails-api/active_model_serializers/pull/1255) Make more class attributes inheritable(@bf4)
```
Comment (bf4) [inline CHANGELOG.md]: ` (@bf4)`

**Comment by bf4** [comment]
Merged in 316026e9ce2dd5068bf7e901dfa84466d123bc7a

**Comment by bf4** [comment]
:rainbow: :heart:

**Pull request closed without merging**